rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/HealthApiAcceptance.java

```diff
@@ -0,0 +1,17 @@
+package gov.cms.qpp.conversion.api.acceptance;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static io.restassured.RestAssured.get;
+
+
+@ExtendWith(RestExtension.class)
+class HealthApiAcceptance {
+	@Test
+	@Tag("acceptance")
+	void apiResponsesWithOkStatus() {
+		get("/health").then().statusCode(200);
```

Comment:
Nice!  We should figure out what should be tested with one of these "Acceptance" test versus one of the "Integration" tests like `QrdaRestIntegration.java`.

Reply:
Agreed, we should add some tasking to the backlog to track scenarios that are currently absent or should be converted to acceptance tests.